packages/block-library/src/post-excerpt/edit.js

```diff
@@ -2,16 +2,96 @@
  * WordPress dependencies
  */
 import { useEntityProp, useEntityId } from '@wordpress/core-data';
-import { PlainText } from '@wordpress/block-editor';
+import { useMemo } from '@wordpress/element';
+import { InspectorControls, RichText } from '@wordpress/block-editor';
+import { PanelBody, RangeControl, ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
 
-function PostExcerptDisplay() {
+function usePostContentExcerpt( excerptLength ) {
+	const [ , , { raw: rawPostContent } ] = useEntityProp(
```

Comment:
Why do we have three arguments, is the first one normalized or something like that?

Reply:
The first one is the edited value, yes.